use_cases/models.py

```diff
@@ -44,6 +44,24 @@ def __str__(self):
         return self.name
 
 
+def check_owner_user(instance, user):
+    return user == instance.created_by
+
+
+def check_supervisor_user(instance, user):
+    return user.is_superuser
```

Comment:
are we sure we want to use the superuser here? I'm asking cause at the moment there are only 3 people that can do this action.

Reply:
I'll add a group and we'll change it later if required.